### ISSUE TITLE
Is there an option of editing an existing feed? If we have made a mistake while entering the RSS link and realise it later, there should be an option of deleting or editing the feed URL. Also, is there a way for us to view the URL that has been added? 

### ISSUE THREAD

**Issue by IIHS-Comm** [janastu/iihs_curation_dashboard#270]
For e.g. today, I realised that the feed I had named Urban Environment was pulling in mostly only waste related news. This made me wonder if I have made a mistake and added the Urban Waste URL by mistake. However, when I looked around to edit the source URL, there doesn’t seem to be any options I can use. In Feedly, there is a way for you to edit the source URL. Clicking on the source URL in Feedly also allows you to see the entire URL, which could help us confirm which URL has been added. Can this be done for our curation tool as well?

![seeing feed name_feedly](https://user-images.githubusercontent.com/38202501/40239434-a7e2ec70-5ad3-11e8-8522-647fffdd8e37.JPG)

**Comment by salus-sage** [comment]
Yes, we haven't added edit source feature yet, will discuss about this with
the team.
